workflows/raredisease.nf

```diff
@@ -164,7 +168,7 @@ workflow RAREDISEASE {
         PREPARE_GENOME.out.fasta,
         PREPARE_GENOME.out.fai,
         INPUT_CHECK.out.ch_case_info,
-        ch_target_bed
+        ch_target_bed.bed
```

Comment:
shouldn't this be `ch_target_bed.out.bed`? I guess that you have tested it though and knows that it shouldn't 😅

Reply:
Yeah but actually I'm curious why it doesn't have to include the `.out`

Reply:
Yeah but actually I'm curious why it doesn't have to include `.out`

Reply:
What I have done in this case is that, I have assigned the output from the CHECK_BED subworkflow i.e CHECK_BED.out to ch_target_bed (line 119). So now when I need to access the attributes from the subworkflow I can simply use ch_target_bed.attribute. Or in other words, ch_target_bed.bed is the same as writing CHECK_BED.out.bed.